docs/source/faq.rst

```diff
@@ -0,0 +1,107 @@
+FAQ: frequently asked questions
+-------------------------------
+
+How do I get the data?
+~~~~~~~~~~~~~~~~~~~~~~
+
+Check ``learner.data``.
+
+
+How do I learn more than one value per point?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the `adaptive.DataSaver`.
+
+
+My runner failed, how do I get the error message?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Check ``runner.task.print_stack()``.
+
+
+How do I get a `~adaptiveLearner2D`\'s data on a grid?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use ``learner.interpolated_on_grid()`` optionally with a argument ``n`` to specify the the amount of points in ``x`` and ``y``.
+
+
+Why can I not use a ``lambda`` function?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The functions evaluations are done on an executor.
+Therefore, the ``function`` needs to be serialized (pickled) and send to the executor, ``lambda``\s cannot be pickled.
+Instead you can probably use ``functools.partial`` to accomplish what you want to do.
+
+
+I get "``concurrent.futures.process.BrokenProcessPool``: A process in the process pool was terminated abruptly while the future was running or pending." what does it mean?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+XXX: add answer!
+
+Where can I learn more about the algorithm used?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Read our `draft paper <https://gitlab.kwant-project.org/qt/adaptive-paper/builds/artifacts/master/file/paper.pdf?job=make>`_ or the source code on `GitHub <https://github.com/python-adaptive/adaptive/>`_.
+
+
+How do I run multiple runners?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Check out `Adaptive scheduler <http://adaptive-scheduler.readthedocs.io>`_, which solves the following problem of needing to run more learners than you can run with a single runner.
+It easily runs on tens of thousands of cores.
```

Comment:
I think this question/answer needs more context.

You can absolutely run more than 1 runner at a time in a notebook in the obvious way:

```python
runner1 = Runner(learner1)
runner2 = Runner(learner2)
```

Reply:
The problem I want to address is when the learner overhead is too large, such that one needs multiple runners in different threads/processes.

For example, when you use a `BalancingLearner` with 10.000 learners.